emulated_hue/api.py

```diff
@@ -9,6 +10,7 @@
 from typing import Any, AsyncGenerator, Optional
 
 import emulated_hue.const as const
```

Comment:
move the emulated_hue.const import

use isort to auto fix it.

Reply:
Believe it or not, precommit moved it there. I used pycharm to sort it and it moved it to the way you would think, but precommit decided to move it back to there.

Reply:
hmmm strange. maybe some setting in isort (which is used by pre commit) ?